<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Export</class>
 <widget class="QWidget" name="Export">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>.mp4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>.avi</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Academy Engraved LET</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>视频格式</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Academy Engraved LET</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>视频分辨率</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>100</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>640*480</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>720*576</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1080*720</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Academy Engraved LET</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>视频码率</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_3">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>170</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>1500k</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3500k</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5000k</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>230</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Academy Engraved LET</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
